clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/config/GoogleConfigurationProperties.groovy

```diff
@@ -30,6 +30,7 @@ class GoogleConfigurationProperties {
     String accountType
     String project
     boolean alphaListed
+    boolean httpLoadBalancingEnabled // TODO(jacobkiefer): Feature flag for L7 development.
```

Comment:
I would like a sanity check to make sure the L3 behavior is the same with and without L7 alongside.